pkg/secrethub/acl.go

```diff
@@ -316,8 +319,8 @@ func (s accessRuleService) Iterator(path string, params *AccessRuleIteratorParam
 
 // AccessRuleIteratorParams specify parameters used when listing access rules.
 type AccessRuleIteratorParams struct {
-	depth     int  // depth defines the depth of traversal for the iterator, -1 means listing all subdirectories.
-	ancestors bool // ancestors defines whether the iterator should also list access rules of parent directories.
+	depth     *uint  	// depth defines the depth of traversal for the iterator, nil means listing all subdirectories.
```

Comment:
To be able to set a value on these fields, they should be exported.

Reply:
Good point! :+1: